cac-sandbox.yml

```diff
@@ -29,7 +29,1160 @@ jenkins:
             value: DCD-CFTAPPS-SBOX
           - key: AKS_AAT_SUBSCRIPTION_NAME
             value: DCD-CFTAPPS-SBOX
-
+  clouds:
+  - azureVM:
+      azureCredentialsId: "jenkinsServicePrincipal"
+      cloudName: "cnp-azure"
+      deploymentTimeout: 1200
+      existingResourceGroupName: "mgmt-vmimg-store-cnptest"
+      maxVirtualMachinesLimit: 40
+      resourceGroupReferenceType: "existing"
+      vmTemplates:
+      - agentLaunchMethod: "SSH"
+        agentWorkspace: "/opt/jenkins"
+        builtInImage: "Windows Server 2016"
+        credentialsId: "vm_agent_creds"
+        diskType: "managed"
+        doNotUseMachineIfInitFails: true
+        enableMSI: false
+        enableUAMI: false
+        executeInitScriptAsRoot: true
+        existingStorageAccountName: "mgmtvmimgstorecnptest"
+        imageReference:
+          id: "/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783/resourceGroups/cnp-vmimages-sandbox/providers/Microsoft.Compute/images/moj-centos-agent74-20181113164555"
```

Comment:
pretty sure this is the old agent version?
https://github.com/hmcts/cnp-infrastructure-bootstrap/blob/master/sandbox.tfvars#L5

Reply:
do you have a new one?

Reply:
I linked to the newer / current version

Reply:
fixed